chart/templates/deployment.yaml

```diff
@@ -17,9 +17,11 @@ spec:
       containers:
         - name: {{ .Values.name }}
           image: {{ printf "%s:%s" .Values.image.repository .Values.image.tag | quote }}
-          command:
-          - {{ .Values.name }}
           imagePullPolicy: Always
+          {{- if ne .Values.secretNameTemplate "" }}
+          args:
+            - --secret-name-template={{ .Values.secretNameTemplate }}
```

Comment:
what is this exactly for?

Reply:
This is the go template I talked about above. The default one is `{{.LastPass.ObjectMeta.Name}}-{{.LastPassSecret.ID}}` in order to avoid breaking changes. Setting the `secretNameTemplate` value in heml chart allows to override this value.